### DISCUSSION TITLE
Donation

### DISCUSSION THREAD

**Discussion by powolnymarcel** [qruto/laravel-wave#6] [Q&A] [answered]
Hello,

I think that the dev community is open & against war. Of course we are ! As we are always trying to find solutions.... That's why I started this small discussion in regard to what is happening in Ukraine. Of course, we all have our daily life's problems and can't help every single person in the world and why this guy but not this one, ... but If some of you can help it will always be an `i++`

Have some of you managed to help this pal  [Slava Razum](https://github.com/slavarazum) by making a donation ? 
 
I tried, but PayPal rejected the donation. 
Saw also that this guy need a smartphone for his grandma. I have a spare one that I don't use. I can send it. But I don't want it to be stolen by the orcs, any safe way of sending it ? 

@slavarazum send me your address and directives to marcel   @ powolny .be 

PS: Never heard, never seen  about @slavarazum before, I just thought "he is a developer as I do and lived a normal life before"

**Comment by slavarazum** [comment, marked as answer]
Hi Marcel!
I am very encouraged to read your message. You are the first person who has responded so succinctly and sincerely.
Looks like we have some problems with ko-fi service and PayPal payments.
Now I'm trying to setup GitHub sponsorships through the Open Collective fiscal host, because Ukrainian bank accounts has't supported yet.
My grandma now in France as a refugee, I can send you her coordinates.

Thank you for your kind words, you made my day 🤝

**Reply by powolnymarcel** [reply]
Of course feel free to send it to my email address. I will send a Samsung Android smartphone tomorrow, I can set it up in Ukrainian language :) but I am afraid I will be stuck in the menu afterwards 😅.

**Reply by slavarazum** [reply]
Hah, I can imagine how our language looks to you 😄 Don't worry about that, there are volunteers who can help her with that 😇

**Comment by slavarazum** [comment]
Thanks to Open Collective, I was finally able to launch a sponsorship profile.
https://github.com/sponsors/qruto
Recent life story, current work, future plans described on the page.
